movie_REST_API/movie_REST_API/ViewModel/MovieViewModel.swift

```diff
@@ -47,41 +75,48 @@ final class MovieViewModel: MovieViewModelProtocol {
             return .defaultColor
         }
     }
-    
+
     func fetchImage(imageURLPath: String, completion: @escaping (Result<Data, Error>) -> ()) {
-        imageService?.fetchImage(imageURLPath: "\(ImageNetworkService.Constant.firstPartURLString)\(imageURLPath)",
-                                 completion: { result in
-            switch result {
-            case let .success(data):
-                completion(.success(data))
-            case let .failure(error):
-                completion(.failure(error))
+        imageService?.fetchImage(
+            imageURLPath: "\(ImageNetworkService.Constant.firstPartURLString)\(imageURLPath)",
```

Comment:
```suggestion
            imageURLPath: imageURLPath
```
ImageNetworkService.Constant.firstPartURLString это внутри сервиса и только для него